app/views/quotes/_form.html.erb

```diff
@@ -2,5 +2,11 @@
   <%= f.input :quoted_person %>
   <%= f.input :location %>
   <%= f.input :body %>
+  Please format as shown below:
+  <div>
+    <span class="quote_example">Quoted Person</span>
```

Comment:
You probably could have put this class on the div and then had:

```scss
.quote-example span {
  font-style: italic;
}
```
Since from this, it appears you want all spans in this block to have italic text. You may also be able to kill the `span` inside the `<q>` and say you want all `<q>` with italic font as well.